### ISSUE TITLE
[Bug]: Nginx doesn't display favicon.ico

### ISSUE THREAD

**Issue by Benisy** [beyondcode/herd-community#830]
### Platform

Windows

### Operating system version

Windows 11 Pro 23H2 22631.3737

### System architecture

Windows

### Herd Version

1.7.2

### PHP Version

8.3.8

### Bug description

If you put this in the head on the site:
`<link href="{{ asset('favicon.ico') }}" rel="shortcut icon" type="image/x-icon" />`

The icon won't display, but the header will have this:
```
HTTP/1.1 302 Moved Temporarily
Server: nginx/1.25.2
Date: Fri, 21 Jun 2024 10:25:08 GMT
Content-Type: text/html
Content-Length: 145
Connection: keep-alive
Location: C:/Program Files/Herd/resources/app.asar.unpacked/resources/valet/server.php
```

**Comment by zainalhassan** [comment]
Im getting the same thing
![Screenshot 2024-07-02 105247](https://github.com/beyondcode/herd-community/assets/67594590/18868091-b185-4644-9909-a6fbc9fa80e5)
![image](https://github.com/beyondcode/herd-community/assets/67594590/d30d1766-bb3d-45f6-b045-ae8313e07088)

**Comment by sschlein** [comment]
Fixed in 1.10.0. If you have a site that's isolated or secured with a local TLS certificate, please go to the conf file at `%USERPROFILE%\.config\herd\config\valet\Nginx` and remove the following lines and restart nginx:

```
    location = /favicon.ico { access_log off; log_not_found off; }
    location = /robots.txt  { access_log off; log_not_found off; }
```
